config/releases.exs

```diff
@@ -31,6 +31,15 @@ defmodule Environment do
       _ -> nil
     end
   end
+
+  def get_local_url(key) do
+    url = get(key)
+
+    case URI.parse(url).host do
+      host when host in ~w(localhost 127.0.0.1) -> url
```

Comment:
Should we allow `0.0.0.0` and `host.docker.internal`?

Reply:
You're right, they should probably be allowed as there is no risk of doing so anyways. What do others think about this?